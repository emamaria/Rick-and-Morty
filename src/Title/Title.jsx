
export const Title = () => {
  return (
    <h1 data-testid="title">Rick And Morty</h1>
  )
}
