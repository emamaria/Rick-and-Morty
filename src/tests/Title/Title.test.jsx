/* eslint-disable no-undef */
import { render} from "@testing-library/react";
import {Title} from "../../Title/Title"

describe('test <Title/>', () => {
    

    test('should match the snapshot', () => {
       
        const {container} = render(<Title/>)
        expect(container).toMatchSnapshot()
    })

    
});
