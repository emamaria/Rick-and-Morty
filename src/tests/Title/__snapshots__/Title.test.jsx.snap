// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <Title/> should match the snapshot 1`] = `
<div>
  <h1
    data-testid="title"
  >
    Rick And Morty
  </h1>
</div>
`;
